alexa/__init__.py

```diff
@@ -28,8 +28,9 @@ def alexa_etl():
 
 
 def top_list(num=100):
-    a = alexa_etl()
-    return [a.next() for x in xrange(num)]
+    return list(
+            alexa_etl()
+    )[:num]
```

Comment:
What problems did you have previously?  Not sure how big alexa_etl is, does it max out at 100?

Reply:
Yes, i try with a big number like 1000000 and actualy it's just 801905 😃, i don't know how top-1m.csv.zip file was created. With the above code, other programs import the alexa lib will not be stop suddenly when it try a big number 😃